test/unit_test_001.cpp

```diff
@@ -83,6 +86,50 @@ unittest(test_fletcher64)
 }
 
 
+unittest(test_FF_1)
+{
+  uint8_t x[1]  = { 0xFF };
+  uint16_t y[1] = { 0xFFFF };
+  uint32_t z[1] = { 0xFFFFFFFF };
+  
+  assertEqual(0, fletcher16(x, 1));
+  assertEqual(0, fletcher32(y, 1));
+  assertEqual(0, fletcher64(z, 1));
+}
+
+
+unittest(test_FF_2)
+{
+  uint8_t x[1]  = { 0xFF };
+  uint16_t y[1] = { 0xFFFF };
+  uint32_t z[1] = { 0xFFFFFFFF };
+
+  Fletcher16 FL16;
+  FL16.begin();
+  assertEqual(0, FL16.getFletcher());
+  FL16.begin(0xFF, 0);
+  assertEqual(0, FL16.getFletcher());
+  FL16.add(0xFF);
+  assertEqual(0, FL16.getFletcher());
```

Comment:
On `defined(ARDUINO_ARCH_SAMD) || defined(ESP32) || defined(ESP8266)` we should have now `_s1 = 0xFF`.

What do you want to test here?

If you would repeat `FL16.add(0xFF);` once more, the bit shift would change `_s1`.

Reply:
> If you would repeat FL16.add(0xFF); once more, the bit shift would change _s1.

Makes sense as an extra test.

Reply:
> What do you want to test here?

That a value of 0xFF is set to (essentially  same as) 0.

Reply:
I added this additional test for all platforms in #12.